src/k8s-engine/wsl.ts

```diff
@@ -1328,11 +1377,16 @@ export default class WSLBackend extends events.EventEmitter implements K8s.Kuber
     try {
       this.setState(K8s.State.STOPPING);
 
-      this.lastCommandComment = 'Stopping Kubernetes';
+      this.lastCommandComment = 'Shutting Down...';
       await this.progressTracker.action(this.lastCommandComment, 10, async() => {
         if (await this.isDistroRegistered({ runningOnly: true })) {
-          await this.execCommand('/usr/local/bin/wsl-service', 'k3s', 'stop');
-          await this.execCommand('/usr/local/bin/wsl-service', '--ifstarted', 'docker', 'stop');
+          if (this.#enabledK3s) {
+            await this.execCommand('/usr/local/bin/wsl-service', 'k3s', 'stop');
```

Comment:
> Also some people seem to want to start without k3s, and then manually go in and start it later for some reason…

Where did you see that? That is not really a valid use case: if you start RD the first time without k3s, then there won't be any k3s binaries in the VM. Or even if there are, if you are using `containerd`, then you are using the Alpine version and not the k3s version. So you can't just start k3s and expect your images to still be there (we would have to connect to the system containerd, and maybe we should always do that, but that is yet another discussion).

Reply:
That was [in Slack](https://rancher-users.slack.com/archives/C0200L1N1MM/p1645031367258209) (not copying here because it's not really relevant); but yeah I agree we don't support doing that, and if _do_ want to do that eventually we should just make the checkbox in the UI do the appropriate things.

Reply:
We should still write the files all the time and call `wsl-service --ifstarted X stop`…